content/200-concepts/100-components/02-prisma-client/054-middleware/200-logging-middleware.mdx

```diff
@@ -51,36 +51,31 @@ generator client {
 }
 
 datasource db {
-  provider = "mysql"
+  provider = "postgresql"
   url      = env("DATABASE_URL")
 }
 
 model Post {
-  authorId  Int?
-  content   String?
   id        Int     @id @default(autoincrement())
-  published Boolean @default(false)
   title     String
-  user      User?   @relation(fields: [authorId], references: [id])
-  language  String?
+  content   String?
+  published Boolean @default(false)
 
-  @@index([authorId], name: "authorId")
+  user     User? @relation(fields: [authorId], references: [id])
+  authorId Int?
 }
 
 model User {
-  email           String  @unique
-  id              Int     @id @default(autoincrement())
-  name            String?
-  posts           Post[]
-  extendedProfile Json?
-  role            Role    @default(USER)
-}
+  id    Int     @id @default(autoincrement())
+  name  String?
+  email String  @unique
 
-enum Role {
-  ADMIN
-  USER
-  MODERATOR
+  posts Post[]
 }
```

Comment:
Why this change to the logging middleware?

Reply:
The intention was to simplify the data model being used by removing "unnecessary" fields and make it consistent with the default data model we use for the docs. I would by happy to undo this change if you don't think its a necessary change.

Reply:
I've restored the model used here. I'll simplify the model in a separate PR